ebl/v3/EBL_v3.0.0.yaml

```diff
@@ -88,33 +87,61 @@ paths:
             ```
             {
               shippingInstructionsReference: 'si-123',
               shippingInstructionsStatus: 'RECEIVED'
             }
             ```
-        4. If a `201` (Created) is returned - all '_relevant_' subscribers will be notified via a [Shipping Instructions Notification](#/ShippingInstructionsNotification). If a `202` (Accepted) is returned a [Shipping Instructions Notification](#/ShippingInstructionsNotification) will be sent once the payload has been processed
-
-        For `POST` `Shipping Instructions` the process ends here.
-          - For response code `201` (Created) the `Shipping Instructions`:
-            - is now stored in the provider system
-            - has status `RECEIVED`
-            - awaits further processing by the provider
-            - a `201` (Created) response is sent to the consumer with a payload containing the `shippingInstructionsReference` and `documentStatus=RECEIVED`
-            - '_relevant_' subscribers have been notified.
-          - For response code `202` (Accepted) the `Shipping Instructions`:
-            - is now accepted in the provider system
-            - has status `RECEIVED` but the status cannot be queried (no `GET` request is possible until the `Shipping Instructions` is further processed in the provider system)
-            - a `202` (Accepted) response is sent to the consumer with a payload containing the `shippingInstructionsReference` and `documentStatus='RECEIVED'`
-            - awaits further processing by the provider
-
-        The provider wil now start asynchronous processing. Once processed, the new status of the `Shipping Instructions` will be communicated via a `Shipping Instructions Notification`. In case the consumer does not subscribe to notifications it is necessary for the consumer to poll on the
+        4. All '_relevant_' subscribers will be notified via a [Shipping Instructions Notification](#/ShippingInstructionsNotification).
+
+        For `POST` `Shipping Instructions` responding with `201` (Created) the process ends here. The `Shipping Instructions`:
+          - is now stored in the provider system
+          - has status `RECEIVED` and the status can be queried (a subsequent `GET` request will retrieve the payload)
+          - a `201` (Created) response is sent to the consumer with a payload containing the `shippingInstructionsReference` and `shippingInstructionsStatus='RECEIVED'`
+          - '_relevant_' subscribers have been notified.
+          - awaits further processing by the provider
+
+        The provider will now start asynchronous processing. Once processed, the new status of the `Shipping Instructions` will be communicated via a [Shipping Instructions Notification](#/ShippingInstructionsNotification). In case the consumer does not subscribe to notifications it is necessary for the consumer to poll on the
+
+            GET /v3/shipping-instructions/{documentReference}
```

Comment:
Do we need to highlight the endpoints?

Reply:
Indenting with 4 spaces should mark it as `code`...